src/psykeco/littlejonh/LittleJonH.java

```diff
@@ -6,7 +6,6 @@
 
 import psykeco.littlejonh.constants.CronExprOrder;
 import psykeco.littlejonh.utility.LittleJonHUtils;
-import sun.security.util.Length;
```

Comment:
This unused import is not available in Java 11

Reply:
is a typo, some automatic import wrong